template_test.go

```diff
@@ -2,13 +2,77 @@ package main
 
 import (
 	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 	"reflect"
 	"strings"
 	"testing"
 	"text/template"
 )
 
+// getTmpFile creates tmp file with a given content and returns its absolute path, relative path and function to remove the file
+func getTmpFile(t *testing.T, content string) (string, string, func()) {
+	tmpFile, err := ioutil.TempFile("", "test-file")
+	if err != nil {
+		t.Fatal(err)
+	}
+	abs, err := filepath.Abs(tmpFile.Name())
+	if err != nil {
+		t.Fatal(err)
+	}
+	wd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	rel, err := filepath.Rel(wd, abs)
+	_, err = tmpFile.WriteString(content)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return rel, abs, func() {
+		path := abs
+		err := os.Remove(path)
+		if err != nil {
+			return
+		}
```

Comment:
I don't see why we need a separate variable or the no-op `if`.

```suggestion
		_ := os.Remove(abs)
```

Reply:
Yep, I was previously handling that error, so this is just a leftover from some refactoring.
Accepting the suggestion :+1: